truss/config/trt_llm.py

```diff
@@ -70,6 +70,14 @@ class TrussTRTLLMBuildConfiguration(BaseModel):
     kv_cache_free_gpu_mem_fraction: float = 0.9
     num_builder_gpus: Optional[int] = None
 
+    @field_validator("max_beam_width", mode="after")
+    @classmethod
+    def check_max_beam_width(cls, v: int):
+        if isinstance(v, int):
+            if v != 1:
+                raise ValueError("max_beam_width must be unary")
```

Comment:
This error can be simpler: 

```
Only `max_beam_width` equal to 1 is supported
```

Reply:
or 
```
max_beam_width > 1 is not supported
```

Reply:
Will change the copy in a follow up PR with similar CLI changes